unit2/unit2.ipynb

```diff
@@ -1284,10 +1284,10 @@
     {
       "cell_type": "code",
       "source": [
-        "push_to_hub(repo_id=\"ThomasSimonini/q-FrozenLake-v1-8x8-slippery\", \n",
-        "                model=model,\n",
-        "                env=env\n",
-        "                )"
+        "repo_id = ",
```

Comment:
Thought it might be better for people to type in their `repo_id` rather than trying to overwrite Thomas' model 😆 .